RNA-seq/01-download_qc_quant.md

```diff
@@ -130,9 +130,8 @@ With this option enabled, Salmon will attempt to correct for the bias that occur
 
 #### `--biasSpeedSamp` 
 
-We'll set this value to `5` to speed up the GC bias correction. 
-This should have very little effect on the results according to the Salmon
-documentation.
+We'll set this value to `5` (the default) to speed up the GC bias correction. 
```

Comment:
Not sure what correct notation is. Feel like this should have a hyphen, but can't always trust feelings.
```suggestion
We'll set this value to `5` (the default) to speed up the GC-bias correction. 
```

Reply:
It is `GC bias` in the Salmon documentation and this [blog post from Mike Love](https://mikelove.wordpress.com/2016/09/26/rna-seq-fragment-sequence-bias/) about alpine.